### ISSUE TITLE
Global suffixes

### ISSUE THREAD

**Issue by src-code** [yahoo/atomizer#91]
Add support for globally-defined suffixes, which will make it easier for common values to be reused across a number of CSS properties.  For example, in config you may specify something like:

```javascript
{
  globals: {
    foo: '10px'
  }
}
```

...which would allow you to write classes like `H-foo` and `P-foo`, which would output:

```css
.H-foo {
  height: 10px;
}
.P-foo {
  padding: 10px;
}
```

...without requiring you to write custom config for every class using the `foo` suffix.

**Comment by src-code** [comment]
This is related to issue #90...

**Comment by 3den** [comment]
:+1: nice looks even better than`P-$foo`. Are we supporting both?

**Comment by thierryk** [comment]
@src-code actually, as @3den mentions it should be:
```css
.H-\$foo {
  height: 10px;
}
.P-\$foo {
  padding: 10px;
}
```

The reason for this is that it is important that authors *recognize* those classes as global variables, hence the addition of the `$` character.

**Comment by src-code** [comment]
So as we discussed offline this morning, Atomizer won't need to offer special support for `$` or any other namespace - it'll be treated the same as any other string value used to denote a global/custom suffix.  It'll be up to a project to define their naming conventions for globals.  We'll be sure to document a recommendation around namespacing and dealing with conflicts like this.

On a side note, what we will want to ensure is that Atomizer properly warns when merging configs that define the same customs/globals but with conflicting values.  This is a need unrelated to the use of a namespace for customs/globals.

**Comment by src-code** [comment]
Addressed in pull #108
